### PR TITLE
CMIS Connector

### PR DESCRIPTION
**Pull request by Nirthika**
## Automation tests
 - Unit tests 
   > N/A
 - Integration tests
   > Four test cases

## Security checks
 > Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
Ran FindSecurityBugs plugin and verified report? yes
Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
- Maven 3.x
- Java 1.8
Platform:
 - UBUNTU 16.04
 - WSO2 EI 6.1.1

### CODE REVIEW

File: src/main/resources/config/component.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ~  under the License.
+-->
+<component name="config" type="synapse/template">
+    <subComponents>
+        <component name="init">
+            <file>init.xml</file>
+            <description>configure cmis connector</description>
+        </component>
+    </subComponents>
+</component>
```
Comment (sivaramya) [inline src/main/resources/config/component.xml]: .xml files should end with a new line. Please add file-ending new line.

File: src/main/assembly/assemble-connector.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly>
```
Comment (sivaramya) [inline src/main/assembly/assemble-connector.xml]: License header is missing

File: src/test/resources/automation.xml [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+                <!--<class>-->
+                <!--<name>org.wso2.carbon.integration.common.extensions.usermgt.UserPopulateExtension</name>-->
+                <!--</class>-->
+                <!--<class>-->
+                <!--<name>org.wso2.esb.integration.common.extensions.axis2server.Axis2ServerExtension</name>-->
+                <!--</class>-->
+            </extentionClasses>
+        </platformExecutionManager>
+
+
```
Comment (kalaiyarasiganeshalingam) [inline src/test/resources/automation.xml]: remove the extra spaces

**Review by hasuniea — APPROVED** [review]

File: src/main/resources/config/component.xml (hunk 1 above)
Comment (Nirthika) [reply src/main/resources/config/component.xml]: Could you please explain why we need to add new line .xml files?

File: src/main/resources/config/init.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+-->
+<template xmlns="http://ws.apache.org/ns/synapse" name="init">
+    <parameter name="cmisUsername" description="Username to log in the Alfresco server"/>
+    <parameter name="cmisPassword" description="Password to log in the Alfresco server"/>
+    <parameter name="cmisInstanceUrl" description="URL of the Alfresco CMIS instance"/>
+    <parameter name="cmisBlockingInvocation" description="If it is true the same thread will serve the response otherwise
```
Comment (hasuniea) [inline src/main/resources/config/init.xml]: Does above line exceed the character length

**Review by hasuniea — APPROVED** [review]

**Review by dimslee — CHANGES_REQUESTED** [review]
Please follow the proper standards for commenting.

http://chris.beams.io/posts/git-commit/

You cannot answer "No" to security section of the GitHub template.

**Review by dimslee — APPROVED** [review]

**Pull request merged**